src/gc/config.d

```diff
@@ -186,6 +186,47 @@ body
     return true;
 }
 
+bool parse(T:const(char)[])(const(char)[] optname, ref const(char)[] str, ref T res)
+in { assert(str.length); }
+body
+{
+
+    int start;
+    int nscanned;
+
+    for(;start<str.length; start++)
+    {
+        if(!isspace(str[start]))
+            break;
+    }
+
+    for(nscanned=start; nscanned<str.length; nscanned++)
+    {
+        if(isspace(str[nscanned]))
+            break;
+    }
+
+    switch(str[start .. nscanned])
+    {
+        case "manual":
```

Comment:
Let's rather use an if-else-ladder that switch/case, b/c the latter requires quite some runtime machinery, and the runtime isn't yet initialized.